composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "amabnl/amadeus-ws-client",
+  "name": "darkmatus/amadeus-ws-client",
```

Comment:
Can you revert this please?

Reply:
Sorry, that change should not be in the request. I revert that.